.kokoro/completeness-check.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+## Get the directory of the build script
+scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+function now() { date +"%Y-%m-%d %H:%M:%S" | tr -d '\n'; }
+function msg() { println "$*" >&2; }
+function println() { printf '%s\n' "$(now) $*"; }
+
+function cleanup() {
+  rm .org-list.txt .new-list.txt .diff.txt
+}
+
+## cd to the parent directory, i.e. the root of the git repo
+cd ${scriptDir}/..
+
+## Locally install artifacts
+#mvn verify -DskipTests
+
+## Find client directory that continas flattened pom and cd into it
+clientDir=$(find -iname ".flattened-pom.xml" | cut -d"/" -f1-2)
```

Comment:
If we plan to templatize this file and roll it out to all clients, we should be able to skip this if flattening is not yet enabled (without failing the build). Since it's possible that we might flatten multiple artifacts, this could be turned into something like:
```
for flattened_pom in `find -name ".flattened-pom.xml"`
do
  # check each ${flattened_pom}
done
```

Then if there are no flattened poms, nothing happens and the check passes.

Reply:
Done.